src/frequenz/core/logging.py

```diff
@@ -0,0 +1,38 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
```

Comment:
2024!

Reply:
We should probably remove these copyright years, from the legal perspective they seem to be useless and we always get them wrong, and it makes testing templates harder, as we need to make the test change every year...